tests/test_config_utils.py

```diff
@@ -131,19 +131,35 @@ def test_autoconfigure_read_both_from_file_and_env(monkeypatch, request):
     from bigchaindb import config_utils
     config_utils.autoconfigure()
 
+    backend = request.config.getoption('--database-backend')
+    database_rethinkdb = {
+        'backend': 'rethinkdb',
+        'host': 'test-host',
+        'port': 4242,
+        'name': 'test-dbname',
+    }
+    database_mongodb = {
+        'backend': 'mongodb',
+        'host': 'test-host',
+        'port': 4242,
+        'name': 'test-dbname',
+        'replicaset': 'bigchain-rs',
+    }
```

Comment:
Might be nice to set some of the variables like `test-host`, `4242`, and `test-dbname` as constants at the start of the test to avoid passing some magic strings around repeatedly.